### PR TITLE
fix ping_hosts test

### LINKED ISSUE #417: ignored test_ping_hosts from base contains Docker logic
need to unignore it and make it independent of Docker operations: extract the mail server setup as part of the CI process / move the flow from base folder

I vote for the first one

### PR DESCRIPTION
**Pull request by maayan-avraham**
fixes #417 
Signed-off-by: Maayan Avraham <maayan.avraham@hpe.com>

### CODE REVIEW

**Comment by maayan-avraham** [comment]
@Bonczidai please review

File: test/io/cloudslang/base/network/example/ping_hosts.inputs.yaml [hunk 1]
```diff
@@ -9,14 +9,10 @@
...
   inputs:
     - ip_list: localhost
-    - docker_host: localhost
-    - docker_port: "49153"
-    - docker_username: root
-    - docker_password: screencast
-    - email_host: localhost
-    - email_port: "25"
+    - hostname: localhost
+    - port: "49154"
```
Comment (Bonczidai) [inline test/io/cloudslang/base/network/example/ping_hosts.inputs.yaml]: I think you should use `hostname` and `port` from the SPF.. that way every mail related input would be assigned from SPF, ping related input would given as input.. WDYT?

**Comment by Bonczidai** [comment]
you can remove `cloud-slang-content\test\io\cloudslang\base\network\example\test_ping_hosts.sl`

File: test/io/cloudslang/base/network/example/ping_hosts.inputs.yaml (hunk 1 above)
Comment (maayan-avraham) [inline test/io/cloudslang/base/network/example/ping_hosts.inputs.yaml]: Agree

**Comment by maayan-avraham** [comment]
@Bonczidai please review again and merge

**Pull request merged**